lib/crawler.js

```diff
@@ -230,42 +237,222 @@ util.inherits(Crawler, EventEmitter);
 Crawler.prototype.start = function() {
     var crawler = this;
 
-    // only if we haven't already got stuff in our queue...
-    crawler.queue.getLength(function(err, length) {
-        if (err) {
-            throw err;
-        }
+    crawler.getRobotsTxt(function (error, robotsTxtUrl, robotsTxtBody) {
```

Comment:
Error should be an error object, methinks (or if you wanna use a string, build an event and emit it instead.)